scribus/ui/fontcombo.h

```diff
@@ -98,4 +98,20 @@ protected slots:
 	bool showLabels;
 };
 
+// based on Qfontcombobox class from Qt project
+class fontFamilyDelegate : public QAbstractItemDelegate
```

Comment:
@khaledhosny  It is Qt way to handle paint a custom widget in any Qt Widgets. Please have a look to[ Model/View Programming in Qt](http://doc.qt.io/qt-5/model-view-programming.html) then have look to [QAbstractItemDelegate](http://doc.qt.io/qt-5/qabstractitemdelegate.html)